android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -54,14 +53,26 @@ fun SurveyQuestionsScreen(
         )
     }
 
+    showExitDialog.let {
```

Comment:
`showExitDialog` is a non-null state, right?

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/122/commits/34df858b7cdc74f4a9252b232270a14b2ef99b47